source/include/FreeRTOS_DNS_Globals.h

```diff
@@ -57,13 +57,6 @@
  * name field is an offset to the string, rather than the string itself. */
     #define dnsNAME_IS_OFFSET    ( ( uint8_t ) 0xc0 )
 
-/** @brief The maximum number of times a DNS request should be sent out if a response
- * is not received, before giving up. */
-    #ifndef ipconfigDNS_REQUEST_ATTEMPTS
-        #define ipconfigDNS_REQUEST_ATTEMPTS    5
-    #endif
-
```

Comment:
Is this set to a default elsewhere in the repo?
Or is this now just an error not to provide it in `FreeRTOSIPConfig.h`?

If it is an error NOT to include this can this check be re-added to `FreeRTOSIP.h`?

Reply:
FreeRTOSIPConfigDefaults.h